2021 01 19 shaun liu pim (#11399)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/unifiedrolemanagementpolicy.md

---
title: "unifiedRoleManagementPolicy resource type"

> description: "A unifiedRoleManagementPolicy specifies the various policies associated with a scope and role definition. It is derived from microsoft.graph.policyBase."

Reviewer: ```suggestion
description: "A unifiedRoleManagementPolicy specifies the various policies associated with a scope and role definition."
```

author: "shauliu"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

# unifiedRoleManagementPolicy resource type

> Namespace: microsoft.graph

Reviewer: Add: [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

A unifiedRoleManagementPolicy specifies the various policies associated with a scope and role definition. It is derived from microsoft.graph.policyBase.

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[List unifiedRoleManagementPolicies](../api/unifiedrolemanagementpolicy-list.md)|[unifiedRoleManagementPolicy](../resources/unifiedrolemanagementpolicy.md) collection|Get a list of the [unifiedRoleManagementPolicy](../resources/unifiedrolemanagementpolicy.md) objects and their properties.|
|[Get unifiedRoleManagementPolicy](../api/unifiedrolemanagementpolicy-get.md)|[unifiedRoleManagementPolicy](../resources/unifiedrolemanagementpolicy.md)|Read the properties and relationships of an [unifiedRoleManagementPolicy](../resources/unifiedrolemanagementpolicy.md) object.|

> |[List effectiveRules](../api/unifiedrolemanagementpolicy-list-effectiverules.md)|[unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md) collection|Get the unifiedRoleManagementPolicyRule resources from the effectiveRules navigation property.|

Reviewer: ```suggestion
|[List effectiveRules](../api/unifiedrolemanagementpolicy-list-effectiverules.md)|[unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md) collection|Get the unifiedRoleManagementPolicyRule resources from the effectiveRules.|
```

> |[List rules](../api/unifiedrolemanagementpolicy-list-rules.md)|[unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md) collection|Get the unifiedRoleManagementPolicyRule resources from the rules navigation property.|

Reviewer: ```suggestion
|[List rules](../api/unifiedrolemanagementpolicy-list-rules.md)|[unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md) collection|Get the unifiedRoleManagementPolicyRule resources from the rules.|
```

## Properties
|Property|Type|Description|
|:---|:---|:---|
|description|String|Description for the policy.|
|displayName|String|Display name for the policy.|
|id|String|Unique identifier for the policy.|

> |isOrganizationDefault|Boolean|This can only be set to true for a single tenant wide policy which will apply to all scopes and roles. Set the scopeId to "/" and scopeType to Directory.|

Reviewer: ```suggestion
|isOrganizationDefault|Boolean|Indicates whether a single tenant-wide policy applies to all scopes and roles. `true` if a single tenant-wide policy applies; otherwise, `false`. Set the scopeId to "/" and scopeType to Directory.|
```

|lastModifiedBy|[identity](../resources/identity.md)|The identity who last modified the role setting.|
|lastModifiedDateTime|DateTimeOffset|The time when the role setting was last modified.|
|scopeId|String|The id of the scope where the policy is created. E.g. "/", groupId, etc.|

> |scopeType|String|The type of the scope where the policy is created. One of Directory, DirectoryRole, Group.|

Reviewer: ```suggestion
|scopeType|String|The type of the scope where the policy is created. Possible values: Directory, DirectoryRole, Group.|
```

## Relationships
|Relationship|Type|Description|
|:---|:---|:---|

> |effectiveRules|[unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md) collection|The list of effective rules like approval rule, expiration rule, etc. evaluated based on inherited referenced rules. E.g. If there is a tenant wide policy to enforce enabling approval rule, the effective rule will be to enable approval even if the polcy has a rule to disable approval.|

Reviewer: ```suggestion
|effectiveRules|[unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md) collection|The list of effective rules, such as approval rule and expiration rule, evaluated based on inherited referenced rules. For example, if there is a tenant-wide policy to enforce enabling the approval rule, the effective rule enables approval even if the policy has a rule to disable approval.|
```

> |rules|[unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md) collection|The collection of rules like approval rule, expiration rule, etc.|

Reviewer: ```suggestion
|rules|[unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md) collection|The collection of rules, such as approval rule and expiration rule.|
```
